packages/api/src/server/permission/service/permission.factory.ts

```diff
@@ -36,7 +36,7 @@ export class PermissionFactory {
     return permissions;
   }
 
-  groupPermission(permissions: Permission[], matchRole: number) {
+  groupPermission(permissions: Permission[], allPermissions: number) {
```

Comment:
can you  please rename `allPermissions` to `allPermissionsLength` xD?

Reply:
yes done 😅